web/src/components/review-form.tsx

```diff
@@ -131,37 +132,42 @@ export default function SwipeReview(props: Props) {
   }, [reviewedSentencesCount, skippedSentencesCount]);
 
   return (
-    <form id="review-form" onSubmit={onSubmit}>
+    <form id="review-form" onSubmit={ onSubmit }>
+      <Prompt
+        when={ reviewedSentencesCount > 0 }
+        message="Reviewed sentences not submitted, are sure?"
+      />
+
       <p>Swipe right to approve the sentence. Swipe left to reject it. Swipe up to skip it.</p>
-      <p>You have reviewed {reviewedSentencesCount} sentences. Do not forget to submit your review by clicking on the &quot;Finish Review&quot; button below!</p>
+      <p>You have reviewed { reviewedSentencesCount } sentences. Do not forget to submit your review by clicking on the &quot;Finish Review&quot; button below!</p>
```

Comment:
Could you revert those white-space changes in this file? While these slipped in in previous changes of mine, I think we should use it without white-spaces in all cases as that's the pattern in the majority of files.

I'll definitely need to look into this (maybe Prettier nicely set up?), that's not something contributors should need to worry about.

Reply:
I'd be fine if you don't want to, as that is an annoying task. I can easily fix that when setting up something more automatic.

Reply:
Don't worry about those, I've created https://github.com/common-voice/sentence-collector/pull/466 which will take care of that once this is merged :)